commands/azureCommands/acr-logs-utils/tableViewManager.ts

```diff
@@ -31,20 +31,18 @@ export class LogTableWebview {
         this.panel.webview.onDidReceiveMessage(async (message: IMessage) => {
             if (message.logRequest) {
                 const itemNumber: number = +message.logRequest.id;
-                try {
-                    await this.logData.getLink(itemNumber).then(async (url) => {
+                await callWithTelemetryAndErrorHandling(
+                    'accessLogFromTable',
+                    async () => await this.logData.getLink(itemNumber).then(async (url) => {
                         if (url !== 'requesting') {
                             await accessLog(url, this.logData.logs[itemNumber].runId, message.logRequest.download);
                         }
-                    });
-                } catch (err) {
-                    const error = parseError(err);
-                    vscode.window.showErrorMessage(`Error '${error.errorType}': ${error.message}`);
-                }
+                    })
+                );
             } else if (message.copyRequest) {
                 // tslint:disable-next-line:no-unsafe-any
                 clipboardy.writeSync(message.copyRequest.text);
-
+                vscode.window.showInformationMessage("The digest was successfully copied to the clipboard.");
             } else if (message.loadMore) {
                 const alreadyLoaded = this.logData.logs.length;
                 await this.logData.loadLogs({
```

Comment:
Can logLogs fail and would you want to show an error message?  If so, you should do the same here, etc.

Reply:
I'll add telemetry to the whole function then.

Reply:
You could, but the telemetry on errors and usage will be less accurate that way.

Reply:
I'd suggest adding message to the telemetry so we can what usage/errors are referring to.
 
```
            await callWithTelemetryAndErrorHandling( 
                'ACR-logsTableListeners', 
                async function (this: IActionContext) => { 
                    this.properties.message = message;
                    ...

Reply:
Ok, I think I got it right this time.